exchange/exchange.go

```diff
@@ -903,6 +903,66 @@ func (e *exchange) getBidCacheInfo(bid *pbsOrtbBid, auction *auction) (cacheInfo
 	return
 }
 
+// getAuctionCurrencyRates returns the constant currency conversion rates that will be used to process this
+// auction. Parameter customRates contains, if any, the bidRequest custom currency conversion rates
+// defined in bidRequest.Ext.  If usePbsRates flag is set to false, all previous Conversions values are discarded
+// and this function merely substitutes Conversions with customRates. But if usePbsRates was ommitted or set to
+// true, customRates' values update or add to the Conversions map
+func (e *exchange) getAuctionCurrencyRates(customRates *openrtb_ext.ExtRequestCurrency) currency.Conversions {
+	// No bidRequest.ext-defined, use PBS rates
+	if customRates == nil || len(customRates.ConversionRates) == 0 {
+		return e.currencyConverter.Rates()
+	}
+
+	var usePbsRates bool = true
```

Comment:
Nitpick: I like how you wrote this in other parts of the code:
```
usePbsRates := customRates.UsePBSRates != nil && *customRates.UsePBSRates
```

Reply:
Does look a lot better. Done